fix-point/src/Refactor.hs

```diff
@@ -0,0 +1,36 @@
+module Refactor where 
+import Data.List 
+
+changeLine :: String -> String -> String 
+changeLine newMod line = case words line of 
+    -- module names 
+    "module" : s : extra -> 
+        unwords ("module": (transform newMod $ splitAt 3 s) : extra)
+
+    -- imports which start with Lib
+    "import" : s : extra -> 
+        if take 3 s == "Lib" then 
+        unwords ("import": (transform newMod $ splitAt 3 s) : extra) 
```

Comment:
There should be no magic constants in code. Better give it name like `libLen` and add documentation what this constant mean. Or just write it as `length "Lib"`.